core/nwb.event.yaml

```diff
@@ -0,0 +1,53 @@
+groups:
+- neurodata_type_def: Events
+  neurodata_type_inc: NWBDataInterface
+  doc: A list of timestamps, stored in seconds, of an event
+  attributes:
+  - name: description
+    dtype: text
+    doc: Description of the event.
+  datasets:
+  - name: timestamps
+    dtype: float
+    dims:
+    - num_events
+    shape:
+    - null
+    doc: Event timestamps, in seconds, relative to the common experiment master-clock stored in 
+      NWBFile.timestamps_reference_time.
+    attributes:
+    - name: unit
+      dtype: text
+      value: seconds
+      doc: Unit of measurement for timestamps, which is fixed to 'seconds'.
+      
+- neurodata_type_def: LabeledEvents
+  neurodata_type_inc: Events
+  doc: A list of timestamps, stored in seconds, of an event that can have different labels. For example,
+    this type could represent the times that reward was given, as well as which of three different 
+    types of reward was given. In this case, the 'label_keys' dataset would contain values {0, 1, 2},
+    and the 'labels' dataset would contain three text elements, where the first (index 0) specifies the
+    name of the reward associated with a label_keys = 0, the second (index 1) specifies the name of the
+    reward associated with a label_keys = 1, etc. The labels do not have to start at 0 and do not need to
+    be sequential, e.g. the 'label_keys' dataset could contain values {0, 10, 100}, and the 'labels' 
+    dataset could contain 101 values, where labels[0] is 'No reward', labels[10] is '10% reward', 
+    labels[100] is 'Full reward', and all other entries in 'labels' are the empty string.
+  datasets:
+  - name: label_keys
+    dtype: int
```

Comment:
If "Values must be 0 or greater, " shouldn't this be `uint`?

Reply:
Now changed.